sponge/src/main/java/net/buycraft/plugin/sponge/logging/BugsnagNilLogger.java

```diff
@@ -0,0 +1,33 @@
+package net.buycraft.plugin.sponge.logging;
+
+import com.bugsnag.Logger;
+
+/**
+ * Created on 3/3/2016 for BuycraftX. Read the copyrights before change the code!
```

Comment:
The logger that BuycraftX uses doesn't have `super.<method>()` statements littered around.

As for the license, it's MIT.